source/Cosmos.System2/FileSystem/Disk.cs

```diff
@@ -23,43 +23,43 @@ public List<ManagedPartition> Partitions
         {
             get
             {
-                var converted = new List<ManagedPartition>();
```

Comment:
The point of this property is to get all of the latest partitions on the disk. If a partition gets deleted, this should also reflect that.